src/renderer/reader/components/Reader.tsx

```diff
@@ -266,19 +205,35 @@ class Reader extends React.Component<IProps, IState> {
         ensureKeyboardListenerIsInstalled();
         this.registerAllKeyboardListeners();
 
+        const store = diReaderGet("store");
+
+        const pubId = store.getState().reader.info.publicationIdentifier;
+        const locator = store.getState().reader.locator;
+        const manifestUrl = store.getState().reader.info.manifestUrl;
```

Comment:
This causes a regression: localhost URL instead of special Readium HTTPS custom protocol. I am looking into it.

Reply:
Fixed via https://github.com/readium/readium-desktop/commit/4e2fe146699bc88e39619d8cc35108f778b54dcd